webapp/src/shared/hooks/HandleRequestError.tsx

```diff
@@ -0,0 +1,21 @@
+import { ResponseError } from '../generated';
+
+export const handleRequestError = (
+  error: unknown,
+  msg: string,
+  logger: (arg0: any) => void = console.error,
+) => {
```

Comment:
question: I see we place the file in the hooks directory, do we plan to change it into a custom hook in the future?

Reply:
We could keep it as a function because we don't use any state or effects. What do you think about moving the file into another directory (maybe `/shared/utils` ?) and changing the extension to `.ts`?

Reply:
Now it should be in the right place